com.github.TuxPusher.yaml

```diff
@@ -0,0 +1,22 @@
+app-id: com.github.TuxPusher
```

Comment:
this app-id is invalid and doesn't meet the requirements. See https://docs.flatpak.org/en/latest/conventions.html#application-ids

Reply:
com.github.mrbid.TuxPusher is better?

Reply:
slightly, the docs says `io.github`, because that the namespace for the associate web pages (whether enabled or not). The linter WILL enforce it.

Reply:
is `tuxpusher.com` for the same project? That could also be an option to have `com.tuxpusher.TuxPusher`

Reply:
Yes it is, I can update it to this.